docs/source/models/cause.rst

```diff
@@ -4,66 +4,119 @@
 Modeling Causes
 ===============
 
-.. todo::
+.. note::
 
-   Overview of modeling GBD causes.
-
-.. contents:
-
-What is a cause?
-----------------
+   Vivarium has the infrastructure to model causes at any level in the GBD hierarchy.
+   However, it is recommended to model GBD causes at the most detailed level to avoid 
+   problems with missingness in the GBD cause hierarchy.
```

Comment:
I recommend active voice, and saying why more specifically:
> If you model a cause at the most-detailed level, there is a better chance of all necessary data being available in GBD.  For example, cause "CKD due to Diabetes" has a relative risk for SBP, but less-detailed cause "CKD" does not have a relative risk for SBP (and we need to derive this RR from the PAF to use this less-detailed cause and this risk factor together in our hypertension treatment model).

Reply:
Thank you Abie for more explicit description!